SetupDataPkg/Docs/Profiles/Overview.md

```diff
@@ -95,18 +80,8 @@ be added, with the old knob being removed as soon as feasible.
 ### Active Profile Selection
 
 The ActiveProfileSelectorLib library class is intended to have the platform override the value. It will retrieve the

```

Comment:
nit: This is not an override, I guess? I thought it should be implemented by the platforms owners?

Reply:
Correct, this language is garbled, I'll fix it.